### DISCUSSION TITLE
German translation reworked and updated [de]

### DISCUSSION THREAD

**Discussion by nutcrapper** [aclap-dev/video-downloadhelper#1072] [Language & Translation]
Hello,

I noticed some irregularities and mistakes, mostly formal ones that still can be corrected.. If you include my corrections, the German translation is probably perfect.
It's 16 lines that I changed, including the 4 lines that still weren't translated. If you have any question to a particular change I have done, ask away. 

[de.txt](https://github.com/user-attachments/files/16448368/de.txt)

**Comment by paulrouget** [comment]
Which version are you using? v9.1 or v9.0 ? I've udpated the german locales in v9.1.

**Reply by nutcrapper** [reply]
Version 9.0.2.12 (stable), target mozilla, locale de.

**Reply by paulrouget** [reply]
Please use version 9.1 beta :) We changed the DE strings there.

https://www.downloadhelper.net/w/Beta-versions

Sorry for the extra work 😅

**Reply by nutcrapper** [reply]
Alright, I fully reworked and updated the translation for 9.1!

[de.txt](https://github.com/user-attachments/files/16451042/de.txt)
[vdh-settings.json](https://github.com/user-attachments/files/16451043/vdh-settings.json)

**Reply by nutcrapper** [reply]
I spotted another mistake:

Correct:

 v9_tell_me_more: 
Mehr erfahren

**Comment by ansgarKf** [comment]
Bon jour,
made further mods ->
[VDH#de.txt](https://github.com/user-attachments/files/16509391/VDH.de.txt).
Would note additional remarks/mods in separate trace.
And ask about several 'quirks'(?!?) in texts.
Like: v9_user_message_no_incognito_body:
You need to turn on that option manually (this is not required).
What to choose: 'You need' or 'not required'?
Sounds strange.
to be continued
A.

**Comment by rudirastlos** [comment]
Very little Correction to German Language
[de.txt](https://github.com/user-attachments/files/19872632/de.txt)
